agent/src/main/java/com/appland/appmap/Agent.java

```diff
@@ -38,8 +39,12 @@ public static void premain(String agentArgs, Instrumentation inst) {
     
     inst.addTransformer(new ClassFileTransformer());
 
-    if (AppMapConfig.load(new File(Properties.ConfigFile)) == null) {
-      Logger.printf("failed to load config %s\n", Properties.ConfigFile);
+    // If the user explicitly specified a config file, but the file doesn't
+    // exist, raise an error. They've almost certainly made a mistake.
+    boolean configMustExist = Properties.ConfigFile != null;
```

Comment:
Calling this `configSpecified` instead would make the next line less confusing. (Java could really use a null coalescing operator...)